### PR TITLE
fts-squat: Removing the plugin

### PR DESCRIPTION
**Pull request by mbettini-ox**

### CODE REVIEW

File: source/configuration_manual/fts/squat.rst [hunk 1]
```diff
@@ -3,11 +3,9 @@
 Squat FTS Engine
 ================
 
-.. deprecated:: v2.1
+.. versionremoved:: v3.0
```
Comment (sirainen) [inline source/configuration_manual/fts/squat.rst:6]: I wonder if we should leave both. Not sure..

File: source/configuration_manual/fts/squat.rst [hunk 2]
```diff
@@ -3,11 +3,9 @@
...
 
-.. warning:: Squat is deprecated and should no longer be used.
-             It is going to be removed in a future version entirely.
-	     It's recommended to use :ref:`fts_backend_solr` instead.
+.. warning:: It's recommended to use :ref:`fts_backend_solr` instead.
```
Comment (sirainen) [inline source/configuration_manual/fts/squat.rst:8]: The sentence no longer sounds like a warning. Maybe it's a "note" now?

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/configuration_manual/fts/squat.rst (hunk 1 above)
Comment (mbettini-ox) [reply source/configuration_manual/fts/squat.rst:6]: I wondered the same, but have found no cases of use of `versionremoved` where`deprecated` has been preserved.

File: source/configuration_manual/fts/squat.rst (hunk 2 above)
Comment (mbettini-ox) [reply source/configuration_manual/fts/squat.rst:8]: here too, the previous cases retained a warning (autocreate-plugin, expire-plugin, both in 2.3.14)
want me to turn them all into notes ?

File: source/configuration_manual/fts/squat.rst [hunk 3]
```diff
@@ -3,11 +3,9 @@
 Squat FTS Engine
 ================
 
-.. deprecated:: v2.1
+.. versionremoved:: 2.4.0;3.0.0
 
-.. warning:: Squat is deprecated and should no longer be used.
-             It is going to be removed in a future version entirely.
-	     It's recommended to use :ref:`fts_backend_solr` instead.
+.. warning:: It's recommended to use :ref:`fts_backend_solr` instead.
```
Comment (cmouse) [inline source/configuration_manual/fts/squat.rst:8]: I  think we could just remove this. People can choose themselves some other fts engine.

**Review by cmouse — APPROVED** [review]

**Pull request merged**